### DISCUSSION TITLE
Add spinner / loading image into dropdown and multiselect component

### DISCUSSION THREAD

**Discussion by st55425** [primefaces/community#1308] [PrimeNG]
Hi, are there any plans to add a spinner and/or loading message when we are using the components for asynchronous requests?
Its 5 years now since the first request of this feature and there is high demand for it.
https://github.com/primefaces/primeng/issues/5052
https://github.com/primefaces/primeng/issues/10038

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for raising the issue but there is already a spinner implemented in virtualScroll lazyload. Is that what you're asking? Please see the example: https://primeng.org/dropdown#lazyvirtualscroll
